packages/react/src/components/UIShell/HeaderMenuItem.js

```diff
@@ -13,13 +13,14 @@ import Link, { LinkPropTypes } from './Link';
 const { prefix } = settings;
 
 const HeaderMenuItem = React.forwardRef(function HeaderMenuItem(
-  { className, children, role, ...rest },
+  { className, isActive, children, role, ...rest },
   ref
 ) {
   return (
     <li className={className} role={role}>
       <Link
         {...rest}
+        aria-current={isActive ? 'page' : null}
```

Comment:
Would we just want them to pass in `<HeaderMenuItem aria-current="page" />` as a prop? Similarly, would we want to use `isActive` to specify a custom selector if they don't want to use the ARIA tech? (Similar to what we do over in breadcrumbs)

Reply:
Just sharing an experience; Using ARIA attribute for a styling purpose has caused a breaking change when we made an a11y structure fix/change.

Reply:
I changed it to match the pattern we use in Breadcrumb. Thanks for the insight @joshblack and @asudoh! Definitely think not wholly on the ARIA attributes is a good way to go here 👍